<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>29</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QPushButton" name="startBtn">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Start the service</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/marche-gui.qrc">
       <normaloff>:/marche/control.png</normaloff>:/marche/control.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopBtn">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Stop the service</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/marche-gui.qrc">
       <normaloff>:/marche/control-stop-square.png</normaloff>:/marche/control-stop-square.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="restartBtn">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Restart the service</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/marche-gui.qrc">
       <normaloff>:/marche/arrow-circle-225.png</normaloff>:/marche/arrow-circle-225.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacker">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="moreBtn">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../res/marche-gui.qrc">
           <normaloff>:/marche/gear.png</normaloff>:/marche/gear.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionShow_output">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/terminal.png</normaloff>:/marche/terminal.png</iconset>
   </property>
   <property name="text">
    <string>Show start/stop output</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/gear.png</normaloff>:/marche/gear.png</iconset>
   </property>
   <property name="text">
    <string>Configure...</string>
   </property>
  </action>
  <action name="actionShow_logfiles">
   <property name="icon">
    <iconset resource="../res/marche-gui.qrc">
     <normaloff>:/marche/blue-documents-text.png</normaloff>:/marche/blue-documents-text.png</iconset>
   </property>
   <property name="text">
    <string>Show logfiles</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/marche-gui.qrc"/>
 </resources>
 <connections/>
</ui>
